src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/RespondToDefenceCallbackHandler.java

```diff
@@ -0,0 +1,72 @@
+package uk.gov.hmcts.reform.unspec.handler.callback;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.Callback;
+import uk.gov.hmcts.reform.unspec.callback.CallbackHandler;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.enums.YesOrNo;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.unspec.callback.CaseEvent.CLAIMANT_RESPONSE;
+
+@Service
+@RequiredArgsConstructor
+public class RespondToDefenceCallbackHandler extends CallbackHandler {
```

Comment:
Will this be only response to full defence type of the defendant response? how about part admit, counter claim, full admit etc?

Reply:
counts for everything, I think this wording makes sense to. The defence is any of those options above. The claimant then responds to that defence

Reply:
Full admission won't be the defence.

Reply:
doesn't mention any differences in the JIRA ticket so for now it is as is.

Reply:
and sure full admission is still a type of defence 😅 but yeah, I have already added a comment to the JIRA ticket about. how this event should handle different defence types as my interpretation is it handles all, but maybe this is not correct

Reply:
The way I read it, it's `Defendant Response` and not the defence 🤔 🤣 
 which has types of `Full Defence`, 'Full Admission`, `Part Admission` and `Counter Claim`.